### ISSUE TITLE
[Dropdown] Multiple selection dropdown should close by clicking text area's empty space

### ISSUE THREAD

**Issue by veke** [Semantic-Org/Semantic-UI#2982]
Multiple selection should close when clicking the text area's empty space liike single selection dropdown does. Now it closes if clicked outside or arrow icon.

**Comment by jlukic** [comment]
Yeah I think that it makes sense that the behavior should be consistent in both cases. I'm just not sure yet whether I think it should close in both, or stay open.

Keep in mind clicking on the toggle arrow will toggle in both cases.
